feature/notification/src/main/java/io/github/droidkaigi/confsched2019/notification/broadcast/OnTokenPublishedBroadcastReceiver.kt

```diff
@@ -0,0 +1,18 @@
+package io.github.droidkaigi.confsched2019.notification.broadcast
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import io.github.droidkaigi.confsched2019.notification.service.FCMService
+
+abstract class OnTokenPublishedBroadcastReceiver : BroadcastReceiver() {
```

Comment:
I have a question. Where is this implementation class? 🙋‍♂️

Reply:
Nothing there. Just a void class for now. It's okay not to implement this.